### PR TITLE
Initial migration for Alembic

### PR DESCRIPTION
**Pull request by pblca**
Alembic configured to use env vars as well

### CODE REVIEW

File: alembic/env.py [hunk 1]
```diff
@@ -18,7 +23,7 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-TARGET_METADATA = None
+TARGET_METADATA = Base.metadata
```
Comment (Salivala) [inline alembic/env.py:26]: pog?

Comment (pblca) [reply alembic/env.py:26]: pog indeed

**Review by Salivala — APPROVED** [review]
looks good. I didn't really do much more than init alembic in the project dir when starting

File: alembic/versions/114909e05e99_.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                    sa.ForeignKeyConstraint(['guild_id'], ['guilds.id'], name='channels_guild_id_fkey'),
+                    sa.ForeignKeyConstraint(['mirror_to_channel_id'], ['channels.id'],
+                                            name='channels_mirror_to_channel_id_fkey'),
+                    sa.PrimaryKeyConstraint('id', name='channels_pkey')
+                    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```
[the remark this answers is not in the document]
Comment (Salivala) [reply alembic/versions/114909e05e99_.py:41]: with sea_orm you'd have to type all this bullshit out manually god fucking bless.

**Pull request merged**